packages/spotlight/src/utils.js

```diff
@@ -172,13 +172,47 @@ function getContainerRect (containerId) {
 	return getRect(containerNode);
 }
 
+function isStandardFocusable (element) {
+	if ((element.tabIndex < 0) || isAtagWithoutHref(element) || isActuallyDisabled(element) || isExpresslyInert(element) || isElementHidden(element)) {
+		return false;
+	} else if ((!element.parentElement) || (element.tabIndex >= 0)) {
+		return true;
+	}
```

Comment:
Please add `else` case to return falsy value for rest conditions.

Reply:
I added else case. Thank you.